src/app/overview/overview.component.html

```diff
@@ -1,34 +1,117 @@
+        Goblint checks that all accesses to shared memory are protected by a common mutex, also taking into account which threads can execute in parallel, and which can't.</p></div>
+    <div class="col-md-3 col-sm-6 mt-5"><h4>Reach safety</h4><p class=""> Goblint is capable of checking whether some assert is guaranteed to hold at a given program point,
+        or whether a certain program is unreachable. This can be used to encode almost any safety property of interest.
+    </p></div>
+    <div class="col-md-3 col-sm-6 mt-5"><h4>Integer Overflows</h4><p class=""> Goblint can verify that programs are free of overflows of (signed) overflows of integers, which
+        are notorious causes of Undefined Behavior and bugs in real-world programs.
+    </p></div>
+    <div class="col-md-3 col-sm-6 mt-5"><h4>And many more...</h4><p class=""> Goblint comes with a wide variety of further analysis, and it is easy to extend for analysis engineers. It suffices
+        to define the domain and the right-hand sides and that's it! Here is a <a href="https://goblint.readthedocs.io/en/latest/developer-guide/firstanalysis/" target="_blank">tutorial</a> for extending Goblint.
+    </p></div>
+    <!-- TODO: fill the feature descriptions
     <div class="col-md-3 col-sm-6 mt-5"><h4>Starvation analysis</h4><p class=""> ...</p></div>
     <div class="col-md-3 col-sm-6 mt-5"><h4>Incremental analysis of Git repositories</h4><p class="">Incremental analysis of Git repositories</p></div>
     <div class="col-md-3 col-sm-6 mt-5"><h4>Fixpoint algorithms</h4><p class=""> Generic fixpoint algorithms with widening and narrowing.</p></div>
     <div class="col-md-3 col-sm-6 mt-5"><h4>Global invariant</h4><p class=""> Global invariant approach and side-effecting.</p></div>
     <div class="col-md-3 col-sm-6 mt-5"><h4>HTML front-end</h4><p class=""> Visualisation of results.</p></div> -->
-    
+
+</div>
+
+<br />
+<br />
+<br />
+
+
+
+
+<div class="row d-flex justify-content-center">
+    <h2>Goblint for...</h2><br>
+</div>
+<div class="row d-flex justify-content-center">
+    <div class="col-md-4 col-sm-6 mt-5"><h4>... software developers.</h4><p class=""> Goblint is free to try out and comes with both a command line interface, the possibility to integrate it into your CI workflow,
+        and an easy to use IDE integration. If your program is open source, feel free to contribute it to our benchmark suite. We're happy to help if there are issues. Give it a try!
+    </p></div>
+    <div class="col-md-4 col-sm-6 mt-5"><h4>... students.</h4><p class=""> We are continuously looking for motivated students who are excited to learn about Static Analysis by Abstract Interpretation and want
+        to contribute to Goblint. This can be in a course, as a HiWi, or as a Bachelor's or Master's thesis, both at UTartu and at TUM. Just reach out to any of us if you are interested!
```

Comment:
HiWi is a very German abbreviation for this. I don't know the specifics, but maybe "as a volunteer (including HiWi)" or something?